optimum/exporters/tasks.py

```diff
@@ -1161,11 +1164,26 @@ def determine_framework(
                 for file in filenames
             ]
         else:
-            if not isinstance(model_name_or_path, str):
-                model_name_or_path = str(model_name_or_path)
-            all_files = huggingface_hub.list_repo_files(model_name_or_path, repo_type="model")
-            if subfolder != "":
-                all_files = [file[len(subfolder) + 1 :] for file in all_files if file.startswith(subfolder)]
+            object_id = model_name_or_path.replace("/", "--")
```

Comment:
Not super familiar with the hub cache, are we sure that replacing "/" by "--" always works?

Reply:
In the "new" caching system I guess yes, I just followed https://github.com/huggingface/huggingface_hub/blob/2c598c484687d586a0ad358301364974912e0262/src/huggingface_hub/file_download.py#L1466-L1467